app/models/card.rb

```diff
@@ -1,4 +1,4 @@
-require 'super_memo'
+require 'super_memo_service'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.